androidApp/src/main/java/com/mbta/tid/mbta_app/android/SheetRoutes.kt

```diff
@@ -27,7 +29,12 @@ sealed class SheetRoutes {
         val tripFilter: TripDetailsFilter?,
     ) : SheetRoutes()
 
-    @Serializable data class RouteDetails(val routeId: String) : SheetRoutes()
+    @Serializable
+    data class RoutePicker(val path: RoutePickerPath, val context: RouteDetailsContext) :
```

Comment:
question: Does the RoutePickerPath only matter if RouteDetailsContext is favorites? Wondering if it can live within `RouteDetailsContext` or if there is a benefit to keeping it separate